app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,20 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name), class: "link" %><br />
```

Comment:
- [ ] The login is not working properly, when I click nothing shows up or changes on the page, whereas this gets back on the console:
![image](https://user-images.githubusercontent.com/62705947/191829934-1104d3ec-f808-416b-b50b-2c21c2d973d4.png)
![image](https://user-images.githubusercontent.com/62705947/191829986-5e780c80-ebe6-4645-8266-874a95d3a707.png)

Reply:
The Log in would not respond if one is not a registered user. The sign-up is what a first time user would need to click, but if one is already signed up, the log in tap works fine for him.

Reply:
The Log in would not respond if one is not a registered user. The sign-up is what a first-time user would need to click, but if one is already signed up, the log in tap works fine for him.

Reply:
The log in tap would not respond if one is not a registered user, instead the Register button would. But if one has registered already, the log in tap would work well for him and link him up to the correct page.